backend/satellite_tools/reposync.py

```diff
@@ -360,14 +360,32 @@ def clear_ssl_cache():
     shutil.rmtree(ssldir, True)
 
 
+def verify_certificates_dates(certs):
+    cert = ""
+    isCert = False
+    valid = True
+    for line in cert_str.split('\n'):
+        if not isCert and line.startswith("-----BEGIN CERTIFICATE"):
+            isCert = True
+        if isCert:
+            cert += line + '\n'
+        if isCert and line.startswith("-----END CERTIFICATE"):
+            valid &= verify_certificate_dates(cert)
```

Comment:
I was wondering if we do really want to return `True` in case passing empty string, or string with no certificate at all. 

With the current proposal these values will pass the validation, what is your expectation here?

```suggestion
            valid = True if valid is None else valid & verify_certificate_dates(cert)
```

Reply:
I think we do it different and abort on the first invalid certificate found.